index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href=".styles.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="main">
+        <header id="header">
+            <div class="logo">
+                <img src="images/tromb.png" alt="picture" id="header-imng" />
+                <nav id="nav-bar">
+                    <ul>
+                        <li><a class="nav-link" href="#features">Features</a></li>
+                        <li><a class="nav-link" href="#How It Works">How It Works</a></li>
+                        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+
+        <div class="form">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+                <input type="email" placeholder="Enter your email address" name="email" id="email" required />
+                <input type="submit" id="submit" value="Get started" />
+            </form>
+        </div>
+
+        <div id="features">
+            <div class="card">
+                <div class="content">
+                    <h2>Premium</h2>
+                    <p>
+                        Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+                    </p>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="content">
+                    <h2>Fast Shipping</h2>
+                    <p>
+                        We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.
+                    </p>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="content">
+                    <h2>Quality Assurance</h2>
+                    <p>
+                        For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.
+                    </p>
+                </div>
+            </div>
+        </div>
+
+        <div id="how-it-works">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;
+           clipboard-write; encrypted-media; gyroscope;
+            picture-in-picture" allowfullscreen>
+        </iframe>
+        </div>
+
+        <div id="pricing">
+            <div class="card">
+                <h2>TENOR TROMBONE</h2>
+                <strong>$600</strong>
```

Comment:
-I would recommend you to style your items in css and avoid using the strong element, css gives you more options to reach your desired outcome of text 😉